src/components/AttachmentModal.js

```diff
@@ -73,6 +74,21 @@ class AttachmentModal extends PureComponent {
         this.submitAndClose = this.submitAndClose.bind(this);
         this.closeConfirmModal = this.closeConfirmModal.bind(this);
         this.isValidSize = this.isValidSize.bind(this);
+        this.getSplitFileName = this.getSplitFileName.bind(this);
+    }
+
+    /**
+     * Returns the filename split into leading and trailing text
+     * @returns {Object}
+     */
+    getSplitFileName() {
+        const fileName = this.props.originalFileName ? this.props.originalFileName : lodashGet(this.state, 'file.name', '');
+        const splittedFileName = fileName.split('.');
+        const trailingText = splittedFileName.pop();
+        const leadingText = splittedFileName.join('.');
+        return {
+            leadingText, trailingText,
```

Comment:
```suggestion
            fileName, fileExtension,
```

Lets edit the above variables same as these names.